src/ctap/data_formats.rs

```diff
@@ -1064,7 +1149,7 @@ mod test {
         let signature_algorithm = SignatureAlgorithm::try_from(&cbor_signature_algorithm);
         let expected_signature_algorithm = SignatureAlgorithm::ES256;
         assert_eq!(signature_algorithm, Ok(expected_signature_algorithm));
-        let created_cbor: cbor::Value = cbor_int!(signature_algorithm.unwrap() as i64);
+        let created_cbor = cbor::Value::from(signature_algorithm.unwrap());
```

Comment:
Nit: it would be more consistent with the rest of the code to do `let created_cbor: cbor::Value = signature_algorithm.unwrap().into();` rather than using `cbor::Value::from`.

Reply:
I can include that in another PR, if you don't like cbor::Value::from.